CMakeLists.txt

```diff
@@ -128,6 +107,22 @@ if(LLVM_ENABLE_ZLIB)
   find_package(ZLIB)
 endif()
 
+#-------------------------------------------------------------------------------
+# Performance configuration
+#-------------------------------------------------------------------------------
+
+include(CheckCXXCompilerFlag)
+include(CheckLinkerFlag)
+add_link_options("$<$<BOOL:${STABLEHLO_ENABLE_LLD}>:-fuse-ld=lld>")
```

Comment:
I found that linking stablehlo-opt to be also much fsater with lld -- it's also recommended in LLVM.

Reply:
Looks like this is a CMake generator expression: https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html
Hadn't seen those before. Need to brush up on CMake :upside_down_face: